alpine-apk/Dockerfile

```diff
@@ -2,15 +2,15 @@
 # https://github.com/openresty/docker-openresty
 
 ARG RESTY_IMAGE_BASE="alpine"
-ARG RESTY_IMAGE_TAG="3.15"
+ARG RESTY_IMAGE_TAG="3.17"
```

Comment:
> Hello, thanks for your contribution.
> 
> My policy is to only bundle the upstream packages with their corresponding Alpine version. This how OpenResty decided to release their software, along with whatever QA they do, etc. So we keep version parity to maintain higher integrity, less edge-bleed.
> 
> If you really want Alpine 3.17 pre-built, there is the built-from-source `alpine` flavor.
> 
> You can see what's available from upstream here: https://openresty.org/package/alpine/
So my use case is to use alpine-apk version so keep docker image slim, but i would like to fix the medium CVEs. 
Also, how frequently is alpine-apk updated?
![image](https://user-images.githubusercontent.com/95269280/229505316-480e4341-7807-41b9-acdd-4c89d468837d.png)

Reply:
Do you mean we cant upgrade openssl 1.1.1t at risk of breaking openresty? I had another query about openresty. I see there is no new release/tag in openresty upstream since 11 months, i will request in parent repo for supporting newer openssl that is not vulnerable !
Also if you can confirm when we can upgrade alpine-apk and if needed i can change tag to 3.18 in my change

Reply:
No, I mean that OpenResty is using `openssl 1.1.1t`. [It is built against that version in the image.](https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile#L15).

But if you go in a shell in that image and `ssh`, it won't necessarily be using that version.

I don't know anything about upstream OpenResty release schedules.   But I will make Alpine 3.18 images once it comes out as stable.

Reply:
Thanks for all this info. last question, when wil alpine-apk be updated next?

Reply:
I don't know, it depends on upstream.